resources/arduino_files/telemetry_board/telemetry_board.ino

```diff
@@ -248,104 +273,66 @@ void Report(unsigned long now) {
 //////////////////////////////////////////////////////////////////////////////////
 // Serial input support
 
-// Accumulates a line, parses it and takes the requested action if it is valid.
-class SerialInputHandler {
-  public:
-    void Handle() {
-      while (Serial && Serial.available() > 0) {
-        int c = Serial.read();
-        if (wait_for_new_line_) {
-          if (IsNewLine(c)) {
-            wait_for_new_line_ = false;
-            input_buffer_.Reset();
-          }
-        } else if (IsNewLine(c)) {
-          ProcessInputBuffer();
-          wait_for_new_line_ = false;
-          input_buffer_.Reset();
-        } else if (isprint(c)) {
-          if (!input_buffer_.Append(static_cast<char>(c))) {
-            wait_for_new_line_ = true;
-          }
-        } else {
-          // Input is not an acceptable character.
-          wait_for_new_line_ = true;
+bool restarting_computer = false;
+IntervalTimer restart_computer_timer(POWER_CYCLE_MS);
+
+void HandleNumEqNum(uint8_t pin_num, uint8_t pin_status) {
```

Comment:
So has the default changed from `1,1` to `1=1`? It looks like the serial input handler does both but you are using the latter version here.

Reply:
No, but I misnamed that method. Should have been HandleNumCommaNum.